README.md

```diff
@@ -88,4 +88,8 @@ To help with the migration, here are the currently known issues with 3.X. This l
 
 ## Security contact
 
-Please disclose any security-related issues or vulnerabilities by emailing [security@swagger.io](mailto:security@swagger.io), instead of using the public issue tracker.
\ No newline at end of file
+Please disclose any security-related issues or vulnerabilities by emailing [security@swagger.io](mailto:security@swagger.io), instead of using the public issue tracker.
+
+### LICENSE
```

Comment:
Sounds like a good idea. Can I suggest more verbose lisense section which is align with how we approach this in other projects?

```md
## License

SwaggerUI is licensed under [Apache 2.0 license](https://github.com/swagger-api/swagger-ui/blob/master/LICENSE).
SwaggerUI comes with an explicit [NOTICE](https://github.com/swagger-api/swagger-ui/blob/master/NOTICE) file
containing additional legal notifications and information.
```

Reply:
Thanks for idea and update @char0n